src/base/bittorrent/session.cpp

```diff
@@ -321,6 +321,11 @@ QString Session::tempPath() const
     return m_tempPath;
 }
 
+QString Session::torrentTempPath(const InfoHash &hash) const
+{
+    return m_tempPath + QString(hash) + "/";
```

Comment:
The users privacy. One can watch the downloaded data (for instance, download to an external hard-drive), but if qBt writes to configuration or state files which are located somewhere else this information and there is a trivial way to avoid it, I think it has to be avoided. qBt may crash leaving a temp file, for example. I know, that one can encrypt or protect those files in other ways, but simple precaution does not cost too much. Then why don't we do that?